packages/components/src/custom/Modal/modal.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, ReactNode } from 'react';
+import {
+  StyledDialog,
+  StyledDialogActions,
+  StyledDialogContent,
+  StyledDialogTitle
+} from '../Dialog';
+
+interface MesheryModalProps {
+  open: boolean;
+  content: ReactNode;
```

Comment:
if content is going to be a react node , we should accept it as children .

Reply:
i agree, but i used react node here, to specify that any type of content can be render here, in case if we pass children here then also we need to specify {children: ReactNode }, to specify that it acn hold any react element

Reply:
ok if this is how it is already used , then great lets stay with that .